Gemfile

```diff
@@ -0,0 +1,16 @@
+# frozen_string_literal: true
+
+source 'https://rubygems.org'
+git_source(:github) { |repo| "https://github.com/#{repo}.git" }
+
+ruby '3.2.2'
+
+# License
+gem 'license_finder', require: false
+
+# Linting
+gem 'ruboclean', require: false
+gem 'rubocop', require: false
+
+# Testing
+gem 'rspec'
```

Comment:
Please keep all gems in the `development' group so that they are not included when using the gem in other projects.
To do so, you can do:
```
gemspec
```
in this file,
and
```
  s.add_development_dependency('rubocop')
  s.add_development_dependency(....
```
in `cfonb.gemspec`